<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>231</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPlainTextEdit" name="plain_1">
           <property name="plainText">
            <string>ABCDEFG
HIJKLMNOP
QRS
TUV
WX YZ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSplitter" name="splitter_7">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QPushButton" name="btn_copy">
            <property name="text">
             <string>btn_copy</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_cut">
            <property name="text">
             <string>btn_cut</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_paste">
            <property name="text">
             <string>btn_paste</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_undo">
            <property name="text">
             <string>btn_undo</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_redo">
            <property name="text">
             <string>btn_redo</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_all">
            <property name="text">
             <string>btn_all</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_zoomIn">
            <property name="text">
             <string>btn_zoomIn</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_zoomOut">
            <property name="text">
             <string>btn_zoomOut</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_clear">
            <property name="text">
             <string>btn_clear</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>511</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QPlainTextEdit" name="plain_2">
           <property name="plainText">
            <string>ABCDEFG
HIJKLMNOP
QRS
TUV
WX YZ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSplitter" name="splitter_11">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QPushButton" name="btn_canPaste">
            <property name="text">
             <string>btn_canPaste</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_firstVisible">
            <property name="text">
             <string>btn_firstVisible</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_blockCount">
            <property name="text">
             <string>btn_blockCount</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_offset">
            <property name="text">
             <string>btn_offset</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_readOnly">
            <property name="text">
             <string>btn_readOnly</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_append">
            <property name="text">
             <string>btn_append</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_insert">
            <property name="text">
             <string>btn_insert</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_plainText">
            <property name="text">
             <string>btn_plainText</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_flags">
            <property name="text">
             <string>btn_flags</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>331</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QPlainTextEdit" name="plain_3">
           <property name="plainText">
            <string>ABCDEFG
HIJKLMNOP
QRS
TUV
WX YZ
</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSplitter" name="splitter_12">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QPushButton" name="btn_cursorRect">
            <property name="text">
             <string>btn_cursorRect</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_move">
            <property name="text">
             <string>btn_move</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_tabFocus">
            <property name="text">
             <string>btn_tabFocus</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_add_10">
            <property name="text">
             <string>btn_add</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_add_107">
            <property name="text">
             <string>btn_add</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_add_108">
            <property name="text">
             <string>btn_add</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_add_109">
            <property name="text">
             <string>btn_add</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_add_110">
            <property name="text">
             <string>btn_add</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_add_111">
            <property name="text">
             <string>btn_add</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>511</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QPlainTextEdit" name="plain_5">
           <property name="plainText">
            <string>ABCDEFG
HIJKLMNOP
QRS
TUV
WX YZ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSplitter" name="splitter_13">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QPushButton" name="btn_center">
            <property name="text">
             <string>btn_center</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_ensure">
            <property name="text">
             <string>btn_ensure</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_doc">
            <property name="text">
             <string>btn_doc</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_mime">
            <property name="text">
             <string>btn_mime</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>310</y>
      <width>331</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QPlainTextEdit" name="plain_6">
           <property name="plainText">
            <string>ABCDEFG
HIJKLMNOP
QRS
TUV
WX YZ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSplitter" name="splitter_14">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QPushButton" name="btn_block">
            <property name="text">
             <string>btn_block</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_cursor">
            <property name="text">
             <string>btn_cursor</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_format">
            <property name="text">
             <string>btn_format</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_merge">
            <property name="text">
             <string>btn_merge</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>231</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_17">
         <item>
          <widget class="QPlainTextEdit" name="plain_4">
           <property name="plainText">
            <string>ABCDEFG
HIJKLMNOP
QRS
TUV
WX YZ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSplitter" name="splitter_15">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QPushButton" name="btn_extra">
            <property name="text">
             <string>btn_extra</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_menu">
            <property name="text">
             <string>btn_menu</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_find">
            <property name="text">
             <string>btn_find</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_18"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1180</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
